dataFiller/src/main/java/com/inctinctools/marathon2017/socmap/datafiller/services/OSMSourceService.java

```diff
@@ -0,0 +1,43 @@
+package com.inctinctools.marathon2017.socmap.datafiller.services;
+
+import com.inctinctools.marathon2017.socmap.data.entities.POI;
+import com.inctinctools.marathon2017.socmap.datafiller.api.Api;
+import com.inctinctools.marathon2017.socmap.datafiller.httpClient.RestClient;
+import com.inctinctools.marathon2017.socmap.datafiller.mapping.Converter;
+import com.inctinctools.marathon2017.socmap.datafiller.model.Element;
+import com.inctinctools.marathon2017.socmap.datafiller.model.ElementsList;
+import enums.POITypes;
+import org.springframework.stereotype.Service;
+import services.POISourceService;
+
+import java.io.UnsupportedEncodingException;
+import java.util.List;
+
+
+@Service
+public class OSMSourceService implements POISourceService {
+
+    private RestClient restClient;
+
+    public OSMSourceService() {
+        restClient = new RestClient();
+    }
+
+    public List<POI> buildQuery(double x1, double y1, double x2, double y2, POITypes type) {
```

Comment:
bad method name - method loads POI from some source - so lets rename it to
getPOI(...)

rename here and in interface